### DISCUSSION TITLE
internal feature importance estimators

### DISCUSSION THREAD

**Discussion by eilyashko** [etna-team/etna#417] [Q&A]
am i wrong or one of the big limitations of using internal feature importance estimators is that they assume no NaN in the data (including exogenous variables?) 

model_feature_selection = DecisionTreeRegressor(random_state=42)

plot_feature_relevance(
    temp_ts,
    relevance_table=ModelRelevanceTable(),
    top_k=30,
    relevance_params={"model": model_feature_selection},
)

**Comment by alex-hse-repository** [comment]
What do you mean under the "limitations"? This method works with NaNs in target/regressor columns, it simply drop them before fitting the model, the only thing is you get the warning about NaNs in your dataset
